src/index.js

```diff
@@ -30,267 +30,277 @@ const sendMessage = async function (opts) {
 }
 
 bot.on('messageCreate', async function (msginst) {
-  let message = msginst.content
-  const userID = msginst.author.id
-  const channelID = msginst.channelId
+  try {
```

Comment:
idk how I feel about wrapping this whole function in a try/catch. wonder if there's a nicer way to do this... i think you can do `bot.on('error', function (err) { ... })` and any uncaught exception from the EventEmitter will end up in there

side note, i'm looking at using https://www.sapphirejs.dev/ for commands and such, haven't investigated its error handling yet.